src/Temperature.cc

```diff
@@ -14,10 +14,14 @@
  * limitations under the License.
  *
 */
+
 #include "ignition/math/Temperature.hh"
 
+#include <istream>
```

Comment:
Are these headers needed?

Reply:
Yes, the `friend ostream` methods are defined in the headers.  You could alternatively use `iostream`.

I have a follow up in the works that will put all the `friend` methods in the `cc` files so that we can use `iosfwd` in the headers.  I didn't want to blow up this PR any larger.